packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -653,6 +653,16 @@ async function handler(
     eventType.schedulingType === SchedulingType.COLLECTIVE ||
     eventType.schedulingType === SchedulingType.ROUND_ROBIN;
 
+  const teamCredentials: Credential[] = [];
```

Comment:
We need to pass `teamCredentials` to the event manager in order to trigger any CRM apps